app/Good_info.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Good_info extends Model
```

Comment:
change model name to GoodInfo and GoodCat, 
add "protected $table =  "tablename"; " to each class to select a table in database
you can see  "https://laravel.com/docs/5.1/eloquent" for more information